test/models/ToolpadEditor.ts

```diff
@@ -131,9 +137,11 @@ export class ToolpadEditor {
     await this.page.mouse.move(
       sourceBoundingBox!.x + sourceBoundingBox!.width / 2,
       sourceBoundingBox!.y + sourceBoundingBox!.height / 2,
-      { steps: 5 },
+      { steps: 10 },
     );
 
+    await expect(sourceLocator).toBeVisible();
```

Comment:
If you're dragging a component from outside the canvas this waits for the page to be visible, otherwise it waits for the canvas. 
I guess the first scenario doesn't make a lot of sense, maybe again what we want here is what `waitForOverlay` is doing?

Reply:
Sorry, nevermind this comment, I'm rereading it now and it's just wrong.